Services/Notifications/classes/Identification/NotificationIdentification.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Notifications\Identification;
+
+/**
+ * @author Ingmar Szmais <iszmais@databay.de>
```

Comment:
You can remove the @author attribute, we have the Git history.

Reply:
As I Approve the reuction of pleonasms i would preserve the auther here since the information may already exists, but not in the project itsself but in its enviroment.

This may be reconsidered when the ILIAS Association sets git as a requirement for the usage of ILIAS but as far as i know its just a strong recommendation at the moment

Reply:
?!? How do you contribute to  the ILIAS code base now, the last 8 years, and in the future **without** Git?. If we move to another VCS in the future, there will be a migration (like done in the past). But for me it's okay, removing this is not a hard requirement.

Reply:
Im not concerned about the contribution, this has definetly a requirement to git, but for the communication in general.
I agree that most profiteers of ILIAS get their product directly from github, but there are some exceptions (with more or less valid reasons) who recieving only the raw data, wich is still a valid distribution since git is only recommended for the installation.
And those profiteers may have not the additional information git providers but should still be able to get in contact with a maintainer or responsible over the code.